### PR TITLE
docs(i18n): fix cp commands to avoid duplicate files

### LINKED ISSUE #11158: cp commands in i18n tutorial create duplicate file (recursing AND globbing)
### Have you read the Contributing Guidelines on issues?

- [x] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Description

Running through the [i18n tutorial](https://docusaurus.io/docs/i18n/tutorial) the `cp` commands under "Translate Markdown Files" were giving unexpected results.

```
cp -r docs/** i18n/fr/docusaurus-plugin-content-docs/current
```

In some shells, mine included, this leads to duplicate files because the shell globs all the files in all the subdirectories first and then recursively copies each one individually to the destination. So I end up with

```
api/file1.md
type/file2.md
file1.md
file2.md
```

Suspect globbing is used at all because in a later command we only want to include md and mdx files, so straight recursion would be too blunt an instrument:

```
cp -r src/pages/**.md i18n/fr/docusaurus-plugin-content-pages
```

Realise these commands are probably meant to be as simple as possible, though they could cause confusion for someone who doesn't twig to -r and ** doing similar things. Would like to hear any thoughts before suggesting a PR to fix. Would it be better to:

1. change the commands to only use recursion OR globbing if possible
2. use other utilities like find or cpio which may look more daunting but work more predictably
3. add a warning message and link to more info "if you see duplicate files in the destination directory"

### Self-service

- [x] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by Rudra-Tiwari-codes**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [x] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #11158) and the maintainers have approved on my working plan.

## Motivation

Fixes #11158

The `cp` commands in the i18n tutorial use both `-r` (recursive) AND `**` (glob pattern), which causes **duplicate files** in some shells where the glob expands all files in subdirectories before the recursive copy runs.

For example, `cp -r docs/**` results in:
```
api/file1.md type/file2.md file1.md file2.md
```

Instead of just the directory structure.

## Test Plan

- Verified the syntax `cp -r docs/.` is POSIX-compliant and works correctly
- The `/. ` syntax copies directory contents recursively without relying on shell globbing
- No duplicate files are created with this approach

### Test links

Deploy preview: https://deploy-preview-____--docusaurus-2.netlify.app/docs/i18n/tutorial#translate-markdown-files

## Related issues/PRs

Closes #11158

Note: Previous PRs #11450 and #11471 attempted to fix this but were closed because the contributors did not sign the CLA.

### CODE REVIEW

**Comment by fransafu** [comment]
**Warning:** This author is forking multiple ML projects such as google-deepmind/alphafold, ml-explore/mlx, openai/CLIP, pytorch/pytorch, tensorflow/tensorflow, anthropics/claude-code, vllm-project/vllm, and others, adding minimal "contributions" (often for tests or miscellaneous changes) without proper validation. A review of their commits shows mostly local implementations of TODOs copied from existing projects, with little to no substantive review or testing.

So far, this author has forked **41 repositories** following the same pattern. Be careful when accepting this PR. It’s also concerning how this author is able to submit PRs across four repositories in the same day, each requiring large context, which strongly suggests a highly automated workflow.

**Comment by Rudra-Tiwari-codes** [comment]
Thank you for the feedback regarding my recent activity. I am a student and I have been using these smaller tasks as a way to familiarize myself with the architecture of various codebases. I was not aware that submitting multiple minor pull requests was considered disruptive to the maintainer workflow or seen as contribution padding. I appreciate the correction and will pay much closer attention to the impact of my work moving forward. I am closing this pull request now to focus on delivering more substantive technical contributions that provide genuine value to the community.
